docker-compose.yml

```diff
@@ -233,6 +233,20 @@ services:
       - "8167:8167"
 
   #
+  # QAI irc bot
+  #
+  faf-qai:
+    container_name: faf-qai
+    build: git://github.com/FAForever/QAI
```

Comment:
Please reference a specific commit or tag, otherwise we never know what we actually get and it might or might not work.